[2025-02-16T08:04:56.271+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: pipeline_group1.extract_data manual__2025-02-16T08:04:53.373217+00:00 [queued]>
[2025-02-16T08:04:56.286+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: pipeline_group1.extract_data manual__2025-02-16T08:04:53.373217+00:00 [queued]>
[2025-02-16T08:04:56.288+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-02-16T08:04:56.289+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2025-02-16T08:04:56.289+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-02-16T08:04:56.314+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-16 08:04:53.373217+00:00
[2025-02-16T08:04:56.326+0000] {standard_task_runner.py:52} INFO - Started process 283 to run task
[2025-02-16T08:04:56.330+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'pipeline_group1', 'extract_data', 'manual__2025-02-16T08:04:53.373217+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_inka_DAG.py', '--cfg-path', '/tmp/tmp044qyoy5', '--error-file', '/tmp/tmpj2lcp9kg']
[2025-02-16T08:04:56.336+0000] {standard_task_runner.py:80} INFO - Job 3: Subtask extract_data
[2025-02-16T08:04:56.449+0000] {task_command.py:371} INFO - Running <TaskInstance: pipeline_group1.extract_data manual__2025-02-16T08:04:53.373217+00:00 [running]> on host 8d7c06d84acb
[2025-02-16T08:04:56.541+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=group1
AIRFLOW_CTX_DAG_ID=pipeline_group1
AIRFLOW_CTX_TASK_ID=extract_data
AIRFLOW_CTX_EXECUTION_DATE=2025-02-16T08:04:53.373217+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-16T08:04:53.373217+00:00
[2025-02-16T08:04:56.572+0000] {base.py:68} INFO - Using connection ID 'postgres_***' for task execution.
[2025-02-16T08:06:03.940+0000] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGKILL
[2025-02-16T08:06:04.650+0000] {taskinstance.py:1412} INFO - Marking task as FAILED. dag_id=pipeline_group1, task_id=extract_data, execution_date=20250216T080453, start_date=20250216T080456, end_date=20250216T080604
[2025-02-16T08:06:07.918+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
